mod-orders-storage/schemas/receiving_history.json

```diff
@@ -21,6 +21,15 @@
       "type": "string",
       "format": "date-time"
     },
+    "checkinItems": {
+      "description": "if true this will toggle the Check-in workflow for details associated with this PO line",
+      "type": "boolean"
+    },
+    "instanceId": {
+      "description": "UUID of the instance record this purchase order line is related to",
```

Comment:
I would propose to indicate the instance id is linked to a piece e.g. `UUID of the instance record this piece is related to`